src/WebSerialLite.h

```diff
@@ -41,45 +41,10 @@ class WebSerialClass {
   void onMessage(RecvMsgHandlerPlus callbackFunc);
   void onError(ErrHandler callbackFunc);
 
-  // Print
+   // Print
```

Comment:
```suggestion
  // Print
```

There seems to be an extra space here.